Examples/Python/GroomUtils.py

```diff
@@ -155,12 +155,16 @@ def FindReferenceImage(inDataList):
     print(" ")
     return inDataList[idx]
 
-def applyResampling(outDir, refFile, inDataListSeg, inDataListImg=[]):
+def applyRigidAlignment(outDir, refFile, inDataListSeg, inDataListImg=[]):
     """
     This function takes in a filelists(binary and raw) and makes the 
     size and spacing the same as the reference
     """
-    print("\n############# Resample #############")
+    isoValue       = 1e-20
+    icp_iterations = 200
+    antialias_iterations = 1 
```

Comment:
I would increase this. Maybe 20 or 30 iterations?

Reply:
Added 2d12d22